### DISCUSSION TITLE
Default compression level for compressor plugins

### DISCUSSION THREAD

**Discussion by 101arrowz** [parcel-bundler/parcel#7087] [Ideas]
The compressor plugins for Gzip and Brotli currently use the default options from `zlib`, but for a production build I would like the files to have the maximum compression available by default, i.e. set `gzipOptions.level = 9` and `brotliOptions.params[BROTLI_PARAM_QUALITY] = BROTLI_MAX_QUALITY`. Also `BROTLI_PARAM_SIZE_HINT` could be set as an optimization.

**Comment by mischnic** [comment]
Should it just always use the maximum compression level? Or should this be user-configurable?

**Reply by 101arrowz** [reply]
For me I'd always use max and I assume it's the same for most.
